content/10-leetcode/00-code/00-py/sliding_window/lc_3318_find_x_sum_all_k_long_subarrays.py

```
@@ -0,0 +1,18 @@
+import pytest
+
+def some_function() -> any:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)